examples/wdio/wdio.mocha.conf.js

```diff
@@ -56,5 +57,5 @@ exports.config = {
     onComplete: function() {
         console.log('that\'s it');
     }
-
-};
\ No newline at end of file
+    
```

Comment:
Haha WebStorm automatically makes sure files end in a newline... I can take the line out after-the-fact on github if you want